great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -209,10 +209,11 @@ def _add_rule_based_profiler_cells(self) -> None:
         self._add_profiler_instructions()
         self.add_code_cell(
             code=f"""\
-suite = context.run_profiler_with_dynamic_arguments(
+result = context.run_profiler_with_dynamic_arguments(
     name="{self._profiler_name}",
-    expectation_suite=validator.expectation_suite
+    batch_request=batch_request,
```

Comment:
Again a modification to use the correct API

Reply:
This was the beginning. Wrong entry point into the API, but we didnt have a test to run the Notebook